src/network/networkd-json.c

```diff
@@ -47,6 +51,24 @@ static int address_build_json(Address *address, JsonVariant **ret) {
         if (r < 0)
                 return r;
 
+        /*
+         * Get T1 and T2 values only when config source is DHCP
+         */
+        if(strcmp(network_config_source_to_string(address->source), "DHCPv6") == 0) {
+                r = dhcp6_lease_get_lifetime(address->link->dhcp6_lease, &lifetime_t1, &lifetime_t2, &lifetime_valid);
```

Comment:
lifetime_valid is not used, just drop it and pass NULL here

Reply:
Fixed it.